paper/symposium.md

```diff
@@ -0,0 +1,214 @@
+available longitudinal studies and a model development framework based
+on the “tidymodels” package in the statistical programming language R
+to evaluate the predictive accuracy of different modeling approaches,
+including logistic regression, elastic-net logistic regression, random
+forests, and neural networks (Kuhn & Wickham, 2019; Kuhn & Johnson,
+2013; Hastie, et al 2009).  The “tidymodels” framework allows for a
+start-to-finish data science pipeline from data wrangling through
+modeling to cross validation of results (Kuhn & Wickham, 2019). In the second stage we
+simulate the predicted impact of different decision-making frameworks
+based on the results of predictions from the first stage, with a
+particular focus on how these results might affect currently
+underserved groups in higher education (Williams et al 2018).
+
+The following panelists will present during the course of the symposium.
+
+*Patricia Martin, Doctoral Student at UCLA* and investigator with the
+Enrollment Management, Recruiting and Access Group, will provide an
+overview of the Unrollment Project, including both the substantive
+goals of the project and the process for undertaking research. The
+Unrollment project has an unusual structure for higher education
+research, where all data and code are publicly available, and research
+can be followed in "real time" as researchers post their results and
+discussions to GitHub, an open repository for code.  The project
+itself has several current key milestones, including the development
+of predictive algorithms for college completion; simulation of
+decision-making based on these algorithms; and calculating likely
+impacts for different groups of students based on these
+milestones. Martin will introduce the use of git and GitHub as a
+model for open-source research collaborations that are available to
+the public in development, and will demonstrate how audience members
+can provide input and feedback to the research team as the work
+progresses. 
+
+*Benjamin Skinner, Assistant Professor of Higher Education and Policy* at
+the University of Florida, will discuss the results of the first stage
+of the project, which involves generating predictive algorithms for
+bachelor's degree completion among recent high school graduates
+attending four-year institutions. The data for this part of the study
+come from the Educational Longitudinal Study of 2002 (Ingels, et al
+2014). Using this data, the project predicts college graduation based
+on a large set of covariates, using four different modeling
+approaches: logistic regression, elastic-net logistic regression,
+random forests and neural nets (Hastie et al, 2009). Skinner will
+summarize the results of these models, including both standard
+measures of model fit and summaries of the substantive
+importance of these different measures of fit.   
+
+*Monique Ositelu, Senior Policy Analyst at the New America Foundation*,
```

Comment:
I really like this paragraph and the focus of Monique's comments. think it will be very engaging for audience members (like me!) that don't know much about predictive algorithms.

I think Monique should go first. this way, there is some broader framing and a hook to pull in the audience and then we get into the meat of how the sausage is made w/ Patricia and Ben

Reply:
Thanks Ozan! I would push back on having me go first for the reason that my institutional profiles builds off of our unrollment work, using the algorithms developed from the unrollment project for my case studies of institutional decision making. Maybe we can add a short sentence that clarifies my analysis compliments (or builds off of) the unrollment project?

Reply:
We'll leave the order as i.